### PR TITLE
Migrate to ukify

### PR DESCRIPTION
**Pull request by DaanDeMeyer**
Let's start using ukify to generate unified kernel images instead of implementing the logic ourselves. This also fixes issues with overlapping PE sections.

### CODE REVIEW

**Comment by DaanDeMeyer** [comment]
This depends on https://github.com/systemd/systemd/pull/25814

**Review by bluca — APPROVED** [review]
lgtm but haven't checked what the test failures are about

File: mkosi/__init__.py [hunk 1]
```diff
@@ -1559,23 +1522,12 @@ def parse_sign_expected_pcr(value: Union[bool, str]) -> bool:
...
         try:
-            # TODO: pyright stumbles over this with
-            # "import_module" is not a known member of module (reportGeneralTypeIssues)
-            # although importlib.import_module exists in Python 3.7
-            # a regular import trips pyflakes, though and I haven't found a way
-            # to silence that
-            importlib.import_module("cryptography") # type: ignore
             return True if shutil.which('systemd-measure') else False
         except ImportError:
             return False
```
Comment (behrmann) [inline mkosi/__init__.py:1527]: `ImportError` cannot occure here anymore.

**Review by behrmann — CHANGES_REQUESTED** [review]
LGTM, but the one try except is not needed anymore.

**Review by behrmann — APPROVED** [review]

**Pull request merged**